hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -487,6 +487,12 @@
                 This is the window size for all health tests.
                 This value is used when entropy is being tested in FIPS/CC compliance mode (for simplicity referred to as FIPS mode).
                 The default value is (2048 bits * 1 clock/4 bits);
+
+                Note that the number of tested bits taken by the conditioner to produce a seed is equal to the window size x 4, except for the startup seed which is produced using the bits of two subsequent windows, i.e., 2 x window size x 4 tested bits.
```

Comment:
It wasn't immediately clear to me what you meant by 2 x window size x 4 tested bits. The x 4 refers to the symbol size right? If so, maybe it would make sense to make this more explicit, what do you think?

Reply:
The `x 4` is really due to the default symbol size of 4 bits but the 4x multiplier is also valid in single-bit mode, i.e. if the symbol size is reduced to 1-bit. Let me try to clarify this.

Reply:
That's now done.